homeassistant/helpers/entity_registry.py

```diff
@@ -511,6 +511,14 @@ def __delitem__(self, key: str) -> None:
         self._unindex_entry(key)
         super().__delitem__(key)
 
+    def get_entity_ids(self) -> ValuesView[str]:
```

Comment:
This isn't used.

Reply:
> There are more places where these can be used, but I only did the device registry cleanup for now

Will be used in a followup PRs when I get some more free cycles to pull apart my integration branches into something consumable

Reply:
They could have been added in the follow up PRs instead, like we always do.

Reply:
Removed in https://github.com/home-assistant/core/pull/112311